csrc/codegen.cpp

```diff
@@ -493,6 +493,19 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
           NVF_ERROR(dtype == DataType::ComplexDouble);
           code_ << "std::complex<double>" << value;
         }
+      } else if (std::holds_alternative<ArrayType>(s->dtype().type)) {
+        NVF_ERROR(value.is<std::vector>());
+        auto dims = static_cast<int>(value.as<std::vector>().size());
+
+        // print out the vector of numbers.
+        code_ << to_str(s->dtype()) << " {";
+        for (auto i = 0; i < dims; i++) {
+          if (i > 0) {
+            code_ << ", ";
+          }
+          code_ << value[i];
```

Comment:
Should we make this section of code a separate function, say, for example `stringify`, probably at `type.cpp` and use `stringify(value[i])`?

Reply:
Sure! I felt like this should have been available in some library somewhere :)

Reply:
A qq, do you mean stringify'ing the vector of values or just that one line `code_ << X'

Reply:
I mean to have a `std::string stringify(PolymorphicValue, DataType)` which recursively calls itself for container types.

Reply:
I see - would it make sense to close this PR, and land the other PR which use Polymorphic Vals, and then I can add this  function to clean it up.